### DISCUSSION TITLE
Is there an update available for Redis?

### DISCUSSION THREAD

**Discussion by kstingel** [leokhoa/laragon#709] [General]
Is there an update available for Redis?
Laragon 6.0.0 ships with `Redis 5.0.14.1`, redis.io lists latest version as `7.2.4` (checked on 2024-03-01)

the only Windows installation information I can find requires using WSL to install, but  can't see how that would update the Redis version available to Laragon

**Comment by RobiNN1** [comment]
There is another redis for windows that is updated but I don't think it is as stable as the old version bundled in laragon 
https://github.com/zkteco-home/redis-windows
